DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -1558,6 +1568,18 @@ protected override void BanPlayer(int playerIndex)
             }
         }
 
+        private void GhostBanIgnoredUser(string playerName)
+        {
+            var user = connectionManager.UserList.Find(u => u.Name == playerName);
+
+            if (user != null)
+            {
+                // Informing the host like we do when we kick might be annoying. So keep it on the downlow.
+                channel.SendBanMessage(user.Hostname, 8);
+                channel.SendKickMessage(user.Name, 8);
```

Comment:
What does 8 mean here?

Reply:
![image](https://github.com/user-attachments/assets/488d2eab-154f-4373-8ea5-75d79d813e26)
The priority of the queued message. Similar to BanPlayer method here.

Reply:
I think at least the arg should be named for clarity (there too). Or could introduce a const, but I am not sure here it is needed.